src/lib/toastr/toastr.service.ts

```diff
@@ -178,9 +178,12 @@ export class ToastrService {
   /**
    * Determines if toast message is already shown
    */
-  isDuplicate(message: string, resetOnDuplicate: boolean) {
+  isDuplicate(message: string, resetOnDuplicate: boolean, countDuplicates: boolean) {
     for (let i = 0; i < this.toasts.length; i++) {
       if (this.toasts[i].message === message) {
+        if (countDuplicates) {
+          this.toasts[i].toastRef.incrementRefCount();
+        }
         if (
           resetOnDuplicate &&
           this.toasts[i].toastRef.componentInstance.resetTimeout
```

Comment:
Will you also remove this second part of the if statement? I left this in while I was initially developing the timeout resets. `ToastrService` shouldn't care if the toast has a `resetTimeout()` as it only talks to `ToastRef`